feat(blog): open-source Experimental Branding Templates (#2053)
mainmatter/mainmatter.com: src/posts/2023-05-24-open-source-experimental-branding-templates.md

---
title: "Open-source Experimental Branding Templates"
authorHandle: davidhajba
tags: [open-source, branding]
bio: David Hajba
description:
  "David Hajba - senior visual / product designer @ Mainmatter created
  Open-source experimental branding templates. In this blog post, he talks about
  the journey & experience"
og:
  image: /assets/images/posts/2023-02-28-ember-template-assert/og-image.jpg
tagline: |

>   <p>For the first hearing, it was surprising and challenging how a visual designer can start open-source projects. As an applied art service provider/ex freelancer, everything started with a client or project requirements and the number of projects that I did for myself just for fun was significantly shrinking, especially as a beginner father. At Mainmatter everyone spends 20% of their time as an active member of the open-source community, so I had to find a solution.</p>

Reviewer: I meant 'when I first heard about this' in Hungary we say 'for the first hearing' maybe it's not a proper translation, what do you think, how would you translate?

>   <p>For the first hearing, it was surprising and challenging how a visual designer can start open-source projects. As an applied art service provider ~ ex freelancer, everything started with a client or project requirements and the number of projects that I did for myself just for fun was significantly shrinking, especially as a beginner father. At Mainmatter everyone spends 20% of their time as an active member of the open-source community, so I had to find a solution.</p>

Reviewer 2: I think you need to provide some context here, explaining that Mainmatter has 20% time which also applies to a designer, etc.

**Update:** I see that comes at the end – I think it'd make sense to reverse the order

Reviewer: @oscard0m 

could you pls update this paragraph;

At Mainmatter everyone spends 20% of their time as an active member of the open-source community. For the first hearing, it was surprising and challenging how a visual designer can start these kinds of projects. As an applied art service provider ~ ex freelancer, everything started with client or project requirements and the number of projects that I did for myself just for fun was significantly shrinking, especially as a beginner father.

Reviewer 2: I think this is a great opportunity for @davidhajba to learn how to do this himself using https://desktop.github.com

Reviewer: I'm happy to learn all new things and improve myself 👌

Please make a hierarchy between animation and desktop.github.com and I will start with it in my 20% time.
I'm super busy in my regular days and these all requires full attention, my private time is absolutely full with family.

Please consider that, I feel physical pain and suffering when I have to code something, that's why I became a visual designer. Despite it would be a terrible torture, I'm not against to learn anything new. Probably it would have negative effect to all my other design works.

Reviewer 2: I'm sure you can work with that – the template is previous blog posts

Reviewer: I saw on screen sharing how Oscar did it and it was way too complex to do it without dev experience.
@oscard0m could you please share your experience too?

I'm happy to start learning animation, although it was not in a focus of my interest, but it's not far from my profession and if you need it of course, let's do it.

Web development is exactly the opposite of my interest and I never said I will take any steps to this direction.

I really hope I can be appreciated for what can I do, which is quite a lot and I do it on high level.
I'm also happy of course to improve myself around my profession.

Reviewer 3: @davidhajba I'll show you on Monday as we discussed, but just so you know, you won't really need any coding knowledge to work on your blog post. It's just like editing a text file really :)

>   <p>For the first hearing, it was surprising and challenging how a visual designer can start open-source projects. As an applied art service provider ~ ex freelancer, everything started with a client or project requirements and the number of projects that I did for myself just for fun was significantly shrinking, especially as a beginner father. At Mainmatter everyone spends 20% of their time as an active member of the open-source community, so I had to find a solution.</p>

Reviewer 2: ```suggestion
  <p>For the first hearing, it was surprising and challenging how a visual designer can start open-source projects. As an applied art service provider/ex freelancer, everything started with a client or project requirements and the number of projects that I did for myself just for fun was significantly shrinking, especially as a beginner father. At Mainmatter everyone spends 20% of their time as an active member of the open-source community, so I had to find a solution.</p>
```

> image: "/assets/images/posts/2023-05-24-open-source-experimental-branding-templates/header-illustration.jpg"

Reviewer 2: let's not mix Mainmatter and any personal/private brands

Reviewer 2: sure, that seems fine 👍

imageAlt:
  "On this cover image, there's a large white geometrical shape of a bug on a
  purple/salmon color geometrical abstract background"
---

## The idea

Among my expertise Branding and Ui/Ux seemed the best direction to start
something with and the idea came quickly, I can be useful for the designer
community using my 20% time for creating inspiration, and free to use Branding -
Ui/Ux Template projects that can be also developed further.

## The journey & experience

After getting used to rules and requirements everywhere at first it was kind of
strange to have all the freedom of the world - I can do anything I want until
the results are interesting and motivating for others.

At this point, I had a voice in my head - that most of the designers would hear

> I assume - ‘Let’s imagine a fictional client or project that I have to find a
> solution for’. Instead of this, I thought what if turn the whole process upside

Reviewer 2: ```suggestion
I assume - "Let’s imagine a fictional client or project that I have to find a
solution for". Instead of this, I thought what if turn the whole process upside
```

down and create something based on my gut feelings, combine surprising elements
together as a montage, that is functional meanwhile, and see what comes out of
it? Why not?

The first concept became boring (I won't show it to anybody) I became a bit
skeptical about my idea. But instead of giving up, I realized what I want.
Extraordinary. Surprising. Anything but boring.

## Concept 2

![Hero Concept 2 Screenshot](/assets/images/posts/2023-05-24-open-source-experimental-branding-templates/concept_2.jpg)

[Preview](https://scene.zeplin.io/project/6455081b1dc67b3c0c397b0e/screen/646f712c6f352921c02462e6)
[Download Sketch file](https://drive.google.com/drive/folders/1Ns0HDHztA8Ki6v8kp8L9NoxP69fI7D-c)

I was searching between Google fonts when I found the most ridiculous and

> strange but well-designed font I've ever seen called ‘Nabla’. I knew I have to

Reviewer 2: ```suggestion
strange but well-designed font I've ever seen called "Nabla". I knew I have to
```

[Download Sketch file](https://drive.google.com/drive/folders/1Ns0HDHztA8Ki6v8kp8L9NoxP69fI7D-c)

Extremely condensed fonts were always something I wanted to try on the web but
never had a chance to do, so Concept 4 seemed just the perfect time for it.
After finding the ‘Rama Gothic C’ font which I considered the most well-designed
in this category, started to test different concepts and how can I make the best
out of it. It turned out a slideshow-style website combines colorful surfaces
with typography playfulness and amazing monochrome photography with
center-aligned overlay color circles, highlighting some important parts of the
image. I will be curious to try further different concepts with the ‘Rama Gothic
C’ font.

## Conclusion

It was very useful for me to experience the freedom of the creation of ‘Branding
templates’ without any rules or requirements and it’s strongly motivating to
suggest braver and more outstanding elements in real-life projects as well. I
hope these templates will be just as much useful and fun to work with - or
implement further - for the designer community, as it was for me. I’d gladly

> offer these as open-source projects.

Reviewer 2: might make sense to put these in a GitHub repo…

Reviewer 3: The files are already in a shared Google Drive folder, I am not sure if we can create a repository with sketch files, but I am also not familiar with what type of files are accepted there :D

---
title: "Open-source Experimental Branding Templates"
authorHandle: davidhajba
tags: [open-source, branding]
bio: David Hajba
description:
  "David Hajba - senior visual / product designer @ Mainmatter created
  Open-source experimental branding templates. In this blog post, he talks about
  the journey & experience"
og:
  image: /assets/images/posts/2023-02-28-ember-template-assert/og-image.jpg
tagline: |

>   <p>For the first hearing, it was surprising and challenging how a visual designer can start open-source projects. As an applied art service provider/ex freelancer, everything started with a client or project requirements and the number of projects that I did for myself just for fun was significantly shrinking, especially as a beginner father. At Mainmatter everyone spends 20% of their time as an active member of the open-source community, so I had to find a solution.</p>

Reviewer 3: ```suggestion
  <p>How can a visual designer work and collaborate on open-source projects? As an applied art service provider/ex-freelancer, my projects always started with a client and project requirements. The number of projects I did for myself just for fun was significantly shrinking, especially since becoming a first-time father. At Mainmatter everyone spends 20% of their time as an active member of the open-source community, so I had to find a way.</p>
```
